web/src/components/njwds-extended/UnStyledButton.tsx

```diff
@@ -1,34 +1,24 @@
 import React, { forwardRef, ReactElement, useEffect, useRef, useState } from "react";
 
 interface Props {
-  style: "default" | "transparentBgColor" | "footerLink";
   className?: string;
```

Comment:
[boulder] How are we accounting for the footer?

Reply:
it's converted to an anchor link. Defining styles by location of the app was a motivating factor behind the refactor; I don't believe footerLink accurately describes what the button is doing if selected.